pom.xml

```diff
@@ -16,22 +16,22 @@
     <!-- Maven plugin versions -->
     <maven.compiler.version>3.13.0</maven.compiler.version>
     <maven.assembly.version>3.7.1</maven.assembly.version>
-    <maven.download.version>1.11.1</maven.download.version>
-    <maven.surefire.version>3.5.1</maven.surefire.version>
+    <maven.download.version>2.0.0</maven.download.version>
+    <maven.surefire.version>3.5.2</maven.surefire.version>
     <jacoco.version>0.8.12</jacoco.version>
 
     <!-- Project dependency version -->
-    <flink.version>1.20.0</flink.version>
-    <kafka.version>3.7.0</kafka.version>
+    <flink.version>1.20.1</flink.version>
+    <kafka.version>3.9.0</kafka.version>
     <slf4j.version>2.0.16</slf4j.version>
-    <log4j.version>2.24.1</log4j.version>
-    <fabric8.kubernetes-client.version>6.13.4</fabric8.kubernetes-client.version>
+    <log4j.version>2.24.3</log4j.version>
+    <fabric8.kubernetes-client.version>7.1.0</fabric8.kubernetes-client.version>
```

Comment:
Has upstream moved to fabric8 v7? I thought that was quite a substantial change...

Reply:
Right, I wasn't sure if aligning the versions would actually cause an issue or not. The test failures look unrelated but I will revert this and see if that unsticks things